detekt-tooling/src/main/kotlin/io/github/detekt/tooling/api/spec/RulesSpec.kt

```diff
@@ -62,6 +62,6 @@ interface RulesSpec {
         /**
          * Run a single rule.
          */
-        class RestrictToSingleRule(val ruleSetId: RuleSet.Id, val ruleName: Rule.Name) : RunPolicy()
+        class RestrictToSingleRule(val ruleSetId: RuleSet.Id, val ruleId: String) : RunPolicy()
```

Comment:
Could this be a value-class that is pre-split, or at least hides the complexity of `split("/", 2).first()` in a `val`?
```suggestion
        class RestrictToSingleRule(val ruleSetId: RuleSet.Id, val ruleId: Rule.Id) : RunPolicy()
```

Reply:
I did these changes at #7468